### DISCUSSION TITLE
Help with hint plugin that do not display correctly on PDF but perfectly in local

### DISCUSSION THREAD

**Discussion by happytodev** [honkit/honkit#397] [Q&A]
I saw that in the generated PDF of JavaScrip ebook example (https://github.com/sumn2u/learn-javascript/blob/main/books/book_en.pdf), that the hint plugin displayed correctly. 

Can you tell if there is something special to do ?

Because for me in pdf, the hint is just a simple text.

The javascript ebook source implementation :
![image](https://github.com/honkit/honkit/assets/425998/5bb6e563-122d-467a-a2c5-9b052b035ae5)

My implementation :

In PDF :
![image](https://github.com/honkit/honkit/assets/425998/e37f7ffd-1702-45b3-8348-a865c4490177)

In local with honkit serve :
![image](https://github.com/honkit/honkit/assets/425998/7660f933-7ce1-4aca-b3f4-4583e647c257)

If you can help me, I would really appreciate it.

Thank you and best regards ;-)
Fred

**Comment by happytodev** [comment]
@azu any idea please ? 🙏

**Reply by azu** [reply]
Is hint plugin [gitbook-plugin-hints](https://www.npmjs.com/package/gitbook-plugin-hints)?
(I have never used this Plugin.)

[gitbook-plugin-hints](https://www.npmjs.com/package/gitbook-plugin-hints) seems to depend on font-awesome and [default-theme](https://github.com/honkit/honkit/blob/fc6092da24f848fe03befe06f03e5c8cc5fc6910/packages/@honkit/theme/src/less/base/alerts.less#L1-L7) implicitly.
This may prevent the icon and background color from being displayed. 

Will the same problem occur when you write the next one?
You can decide whether the plug-in is the cause or the theme is the cause.

```html
<div class="alert alert-info">
Hint text
</div>
```

📝 This plugin will convert `{% hint style='info' %}...{% endhint %}` to `<div class="alert alert-info">...</div>`.

Do you use themes other than default?
The non-default theme may does not includes `.alert` class.
